.github/workflows/c-cpp.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-11.0, ubuntu-20.04]
+        os: [macos-12.0, ubuntu-20.04]
```

Comment:
I assume the upgrade in here is opportunistic, rather than relevant to the change?

Reply:
Yeah. opportunistic